content/docs/command-reference/add.md

```diff
@@ -243,3 +246,44 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+Let's take an example to illustrate how `.dvcignore` interacts with `dvc add`.
+
+```dvc
+$ mkdir dir
+$ echo file_one > dir/file1
+$ echo file_two > dir/file2
+```
+
+Now add `file1` to `.dvcignore` and track the entire `Dir` directory with
+`dvc add`.
+
+```dvc
+$ echo Dir/file1 > .dvcignore
+$ dvc add Dir
+```
+
+Let's now modify `file1`(which is listed in `.dvcignore`) and run `dvc status`.
```

Comment:
Missing a space here. My suggestion in https://github.com/iterative/dvc.org/pull/1346#discussion_r432967483 had the space. Suggestions can be easily applied from Github directly. Or copy-paste the full text. Please avoid re-typing stuff.
Also, let's use column at the end of this sentence:
```suggestion
Let's now modify `file1` (which is listed in `.dvcignore`) and run `dvc status`:
```
Thanks